rocketc/rocketc.py

```diff
@@ -378,3 +379,12 @@ def get_groups(self):
                 list_groups.append(group["name"])
 
         return list_groups
+
+    def _private_channel(self, room_name):
+        url_search = "{}?{}={}".format("channels.info", "roomName", room_name)
+        channel = self._request_rocket_chat("get", url_search)
+        if "channel" in channel and channel["channel"]["t"] == "c":
```

Comment:
It would be nice if you add some comments about the meaning of the keys and values for a channel. For example channel["channel"]["t"] == "c" means that the channel is public?

Reply:
ok